src/routes/chat/Chat.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Moment from 'react-moment';
+
+import ChatList from './ChatList/ChatList.container';
+import ChatRoom from './ChatRoom/ChatRoom.container';
+import ChatInput from './ChatInput/ChatInput.container';
+import ChatOffer from './ChatOffer/ChatOffer';
```

Comment:
All these should be called using their respective `index.js`s and thus don't have the `.container` ending. Also delete the `ChatOffer.container.js` file if it is empty.